cli/index.ts

```diff
@@ -8,17 +9,20 @@ program
 	.name( 'studio' )
 	.description( 'Studio by WordPress.com CLI' )
 	.version( version )
-	.option(
-		'--output-format [format]',
-		'Specify a non-standard output format',
-		( value: string ) => {
-			if ( value !== 'json' ) {
-				throw new Error( 'The only custom output format supported is "json"' );
-			}
-			return value;
-		}
+	.addOption(
+		new Option( '--output-format [format]', 'Specify a non-standard output format' )
+			.argParser( ( value: string ) => {
+				if ( value !== 'json' ) {
+					throw new Error( 'The only custom output format supported is "json"' );
+				}
+				return value;
+			} )
+			.hideHelp()
```

Comment:
This option is only meant for internal use, which is why I updated the code to hide it from the help page.